charts/docker-hub-rate-limit-exporter-chart/ci/all-enabled-values.yaml

```diff
@@ -70,28 +93,30 @@ serviceAccount:
     name: ""
 
 podAnnotations:
-  sidecar.istio.io/inject: "false"
+    sidecar.istio.io/inject: "false"
 
-securityContext:
-  fsGroup: 65534
-  runAsGroup: 65534
-  runAsNonRoot: true
-  runAsUser: 65534
+podSecurityContext:
+    fsGroup: 65534
```

Comment:
I have the suspicion that one of these configuration breaks the CI-Pipeline tests... not sure though

Reply:
Ok, that was it. after commenting those lines the ci pipeline now works. :-)

Reply:
Probably the capability `NET_BIND_SERVICE` is missing. With this capability I think we can run the container as non-root and have it bind (privileged?) ports.

Reply:
Its because the socket can't bind on ports lower than 1024 for unprivileged users.
```
service:
    port: 9100
    targetPort: 9100
```

Reply:
I choose option 1 since 80 and 8080 is generally used by nginx/apache